content/contest/hash.sh

```diff
@@ -0,0 +1 @@
+tr -d '[:space:]' | md5sum
```

Comment:
I think we can get rid of this; hash-cpp.sh should be fine for other file types as well in 99% of cases, and when it isn't (e.g. Python int division accidentally commenting out a line; not that KACTL currently supports Python) we can skip the hash for that file.